plugin/src/main/scala/org/virtuslab/akkasaferserializer/SaferSerializerPlugin.scala

```diff
@@ -0,0 +1,22 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+
+class SaferSerializerPlugin(override val global: Global) extends Plugin {
+  override val name: String = "safer-serializer-plugin"
+  override val description: String = "checks for Behavior[A] and ActorRef[A]"
+  private val pluginOptions = new PluginOptions(false)
+
+  private val gatherTypes = new SaferSerializerPluginComponent(pluginOptions, global)
+  override val components: List[PluginComponent] = List(gatherTypes)
+
+  override def init(options: List[String], error: String => Unit): Boolean = {
+    pluginOptions.verbose = options.exists(_.equals("verbose"))
+    true
+  }
+
+  override val optionsHelp: Option[String] = Some("""
+      |verbose - print additional info about leafs and roots
```

Comment:
Hmmm it's pretty unclear what `leafs and roots` correspond to here

Reply:
I'd either skip it completely (just `print additional diagnostic info`) or provide some clearer explanation (`print additional info about detected serializable classes`?)